controller/controller.go

```diff
@@ -187,6 +188,8 @@ type ProvisionController struct {
 	claimsInProgress sync.Map
 
 	volumeStore VolumeStore
+
+	volumeLister corelisters.PersistentVolumeLister
```

Comment:
Why `volumeLister`? There is `volumeInformer` and `volumes` already available.

Reply:
Looks like i totally missed `volumes`. I've only tested this with an explicit lister so will try that out and update the commit.